### DISCUSSION TITLE
Can somebody please point me to the bit of code that shows ghost autocomplete

### DISCUSSION THREAD

**Discussion by lostmsu** [PowerShell/PowerShell#24374] [Q&A] [answered]
I mean the autocomplete from history given the current prefix that you can select by pressing right arrow:

![image](https://github.com/user-attachments/assets/a0618abb-d57b-44c0-ae97-693984e1c09c)

**Comment by MartinGC94** [comment, marked as answer]
It's handled by PSReadLine, here's the relevant method: https://github.com/PowerShell/PSReadLine/blob/a1130b519001492969fda4c410ffe803b600825e/PSReadLine/Prediction.Views.cs#L1364
